### PR TITLE
Remove Protobuf dependency 

### PR DESCRIPTION
**Pull request by jeremyjiang-dev**

### CODE REVIEW

**Review by visumickey — APPROVED** [review]
Nice and a simple change which is an outcome of the earlier changes. Clean good work.

LGTM!

File: FirebasePerformance.podspec [hunk 1]
```diff
@@ -42,7 +42,7 @@ Firebase Performance library to measure performance of Mobile and Web Apps.
 
   s.public_header_files = base_dir + 'Sources/Public/FirebasePerformance/*.h'
 
-  preprocessor_definitions = 'GPB_USE_PROTOBUF_FRAMEWORK_IMPORTS=1 ' + 'FIRPerformance_LIB_VERSION=' + String(s.version)
+  preprocessor_definitions = 'FIRPerformance_LIB_VERSION=' + String(s.version)
```
Comment (visumickey) [inline FirebasePerformance.podspec:45]: I think line 32 can also be removed.

The below files would not be required anymore.
`'ProtoSupport/**/*.[mh]'`

**Review by paulb777 — APPROVED** [review]
Love to see all this code eliminated! ❤️

**Review by visumickey — APPROVED** [review]

**Comment by visumickey** [comment]
> # Coverage Report
> ### Affected SDKs
> * #### `FirebasePerformance-iOS-FirebasePerformance.framework`
>   SDK overall coverage changed from **75.93%** ([1decaec](https://github.com/firebase/firebase-ios-sdk/commit/1decaecc2ad9d343bc2f928b53c76a78b70fa8fd))  to **92.33%** ([9bbcc00](https://github.com/firebase/firebase-ios-sdk/commit/9bbcc00c14aa22c173dbbe395d1636607aa296e2))  by **+16.41%**.
>   
>   
>   
>   Filename
>   Base ([1decaec](https://github.com/firebase/firebase-ios-sdk/commit/1decaecc2ad9d343bc2f928b53c76a78b70fa8fd))
>   Head ([9bbcc00](https://github.com/firebase/firebase-ios-sdk/commit/9bbcc00c14aa22c173dbbe395d1636607aa296e2))
>   Diff
> 
> ### Test Logs
> * `Base (1decaecc)`: https://github.com/firebase/firebase-ios-sdk/actions/runs/1113012935
> * `Head (9bbcc00c)`: https://github.com/firebase/firebase-ios-sdk/actions/runs/1113358400

Tremendous increase in our code coverage just because of deleting these lines of code. Another feather in hat because of this change.

File: FirebasePerformance.podspec (hunk 1 above)
Comment (jeremyjiang-dev) [reply FirebasePerformance.podspec:44]: Good catch! Done.

**Pull request merged**